lib/inline_svg/transform_pipeline/transformations.rb

```diff
@@ -25,12 +25,25 @@ def self.all_transformations
 
   def self.lookup(transform_params)
     without_empty_values(transform_params).map do |key, value|
```

Comment:
The `without_empty_values` method feels misleading now because it's taking user-supplied values, removing nils *and* merging them with non-nil default values. The name doesn't really tell us what's happening.

This concept might be more clearly expressed using array concatenation. For example:

```ruby
(without_empty_values(transform_params) + all_default_values).map do |key, value|
#etc...
```

What do you think?